plugins/modules/nsxt_logical_routers.py

```diff
@@ -274,21 +273,180 @@ def update_params_with_id (module, manager_url, mgr_username, mgr_password, vali
         logical_router_params['advanced_config']['transport_zone_id'] = get_id_from_display_name (module, manager_url,
                                                                                 mgr_username, mgr_password, validate_certs,
                                                                                 "/transport-zones", transport_zone_name)
+
```

Comment:
Bug ID Fixed - 2634460 - ID should be replace with name for "redundant_uplink_port_ids" under "ha_vip_configs" in "nsxt_logical_routers" module

Added a new field "redundant_uplink_port_names" which will be passed by customer. This field will be used to extract the corresponding IDs and then will be replaced with the original field "redundant_uplink_port_ids".